photon-api/src/main/scala/com/linkedin/photon/ml/function/glm/SingleNodeGLMLossFunction.scala

```diff
@@ -148,7 +148,8 @@ object SingleNodeGLMLossFunction {
    */
   def apply(
       configuration: GLMOptimizationConfiguration,
-      singleLossFunction: PointwiseLossFunction): SingleNodeGLMLossFunction = {
+      singleLossFunction: PointwiseLossFunction,
+      index: Int = -1): SingleNodeGLMLossFunction = {
```

Comment:
Add comment here too.

Reply:
In addition, parameter name and type should change (see below):
```
interceptIndexOpt: Option[Int] = None
```